game/script-topics.rpy

```diff
@@ -161,7 +161,7 @@ label talk_did_you_have_pets:
     else:
         n 1unmbs "What about you,{w=0.1} [player]?"
         menu:
-            n "Did you get another one?"
+            n "Well?"
```

Comment:
Seems to me here like we don't mention what the actual question is. Though I might be just blind